embed/templates/examples/minimal.yaml

```diff
@@ -1,35 +1,306 @@
+  #   io_read_bandwidth_max: "/dev/disk/by-path/pci-0000:00:1f.2-scsi-0:0:0:0 100M"
+  #   io_write_bandwidth_max: "/dev/disk/by-path/pci-0000:00:1f.2-scsi-0:0:0:0 100M"
+
+# # Monitored variables are applied to all the machines.
+monitored:
+  # # The communication port for reporting system information of each node in the TiDB cluster.
+  node_exporter_port: 9100
+  # # Blackbox_exporter communication port, used for TiDB cluster port monitoring.
+  blackbox_exporter_port: 9115
+  # # Storage directory for deployment files, startup scripts, and configuration files of monitoring components.
+  # deploy_dir: "/tidb-deploy/monitored-9100"
+  # # Data storage directory of monitoring components.
+  # data_dir: "/tidb-data/monitored-9100"
+  # # Log storage directory of the monitoring component.
+  # log_dir: "/tidb-deploy/monitored-9100/log"
+
+# # Server configs are used to specify the runtime configuration of TiDB components.
+# # All configuration items can be found in TiDB docs:
+# # - TiDB: https://pingcap.com/docs/stable/reference/configuration/tidb-server/configuration-file/
+# # - TiKV: https://pingcap.com/docs/stable/reference/configuration/tikv-server/configuration-file/
+# # - PD: https://pingcap.com/docs/stable/reference/configuration/pd-server/configuration-file/
+# # - TiFlash: https://docs.pingcap.com/tidb/stable/tiflash-configuration
+# #
+# # All configuration items use points to represent the hierarchy, e.g:
+# #   readpool.storage.use-unified-pool
+# #           ^       ^
+# # - example: https://github.com/pingcap/tiup/blob/master/examples/topology.example.yaml.
+# # You can overwrite this configuration via the instance-level `config` field.
+# server_configs:
+  # tidb:
+  # tikv:
+  # pd:
+  # tiflash:
+  # tiflash-learner:
 
+# # Server configs are used to specify the configuration of PD Servers.
 pd_servers:
-  - host: 172.19.0.101
-  - host: 172.19.0.102
-  - host: 172.19.0.103
+  # # The ip address of the PD Server.
+  - host: 10.0.1.11
+    # # ssh port of TiUP management cluster.
+    # ssh_port: 22
+    # # PD Server name
+    # name: "pd-1"
+    # # communication port between TiDB Server and PD Server.
+    # client_port: 2379
+    # # Communication port between PD Server cluster nodes.
+    # peer_port: 2380
+    # # PD Server deployment file, startup script, configuration file storage directory.
+    # deploy_dir: "/tidb-deploy/pd-2379"
+    # # PD Server data storage directory.
+    # data_dir: "/tidb-data/pd-2379"
+    # # PD Server log file storage directory.
+    # log_dir: "/tidb-deploy/pd-2379/log"
+    # # numa node bindings.
+    # numa_node: "0,1"
+    # # The following configs are used to overwrite the `server_configs.pd` values.
+    # config:
+    #   schedule.max-merge-region-size: 20
+    #   schedule.max-merge-region-keys: 200000
+  # # The ip address of the PD Server
+  - host: 10.0.1.12
+    # # ssh port of TiUP management cluster.
```

Comment:
I think we could only keep comments for the first host in the list, not to make the file too long.